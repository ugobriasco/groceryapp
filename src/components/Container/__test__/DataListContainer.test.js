import React from 'react';
import renderer from 'react-test-renderer';

import { View } from 'react-native';
import { DataListContainer, styles} from '../';

// Note: test renderer must be required after react-native.


it('renders correctly', () => {
  const rendered = renderer.create(<DataListContainer />).toJSON();
  expect(rendered).toBeTruthy();
});

it('styles is an object', () => {
expect(typeof styles).toBe('object');
});

it('renders the child prop', () => {
	const rendered = renderer.create(<DataListContainer><View /></DataListContainer>).toJSON();
	expect(rendered).toMatchSnapshot();
});