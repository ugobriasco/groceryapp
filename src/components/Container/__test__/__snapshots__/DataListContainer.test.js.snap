// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the child prop 1`] = `
<View
  style={undefined}
>
  <View />
</View>
`;
